dts/bindings/input/ilitek,ili2132a.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Joel Jaldemark
+# SPDX-License-Identifier: Apache-2.0
+
+description: ILI2143A capactitive touch controller
+
+compatible: "ilitek,ili2132a"
+
+include: i2c-device.yaml
+
+properties:
+  irq_gpios:
```

Comment:
devicetree propery name use dashes, there's a note somewhere in the doc :-) `irq-gpios` (the one in the code is fine, it's translated to underscore in the headers)